setup/ansible-inventory

```diff
@@ -72,25 +72,38 @@ iojs-release-armv7-wheezy-1
 release-scaleway-debian7-arm-1
 
 [iojs-raspbian]
-iojs-ns-pi1p-1
-iojs-ns-pi1p-2
-iojs-ns-pi1p-3
-iojs-ns-pi1p-4
-iojs-ns-pi1p-5
+release-nodesource_rvagg-debian7-arm_pi1p-1
+release-nodesource_andineck-debian7-arm_pi1p-1
+test-nodesource_andineck-debian7-arm_pi1p-1
+test-nodesource_bengl-debian7-arm_pi1p-1
+test-nodesource_bengl-debian7-arm_pi1p-2
 iojs-ns-pi1p-6
```

Comment:
hosts with old names are ones that haven't been reprovisioned yet and are still working on original setup, I plan on getting to all of them soon